src/views/components/display-html/DisplayHTML.native.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import Html from 'react-native-render-html';
+import { string } from 'prop-types';
+import { Box } from '../../components';
+
+class DisplayHTML extends Component {
```

Comment:
Also, are we able to convert this into a stateless function? I don't think we'll ever need lifecycle methods nor state for this component.